tests/acceptance/features/apiSharingNg/listPermissions.feature

```diff
@@ -347,16 +347,16 @@ Feature: List a sharing permissions
       """
 
   @issues-8352
-  Scenario: sharer lists permissions of a shared project space
+  Scenario Outline: sharer lists permissions of a shared project space
     Given using spaces DAV path
     And user "Brian" has been created with default attributes and without skeleton files
     And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
     And user "Alice" has created a space "new-space" with the default quota using the Graph API
     And user "Alice" has sent the following space share invitation:
-      | space           | new-space    |
-      | sharee          | Brian        |
-      | shareType       | user         |
-      | permissionsRole | Space Viewer |
+      | space           | new-space          |
+      | sharee          | Brian              |
+      | shareType       | user               |
+      | permissionsRole | <permissions-role> |
     And user "Alice" has created the following resource link share:
```

Comment:
```suggestion
    And user "Alice" has created the following space link share::
```
If this is about sharing space I think we have a step for that

Reply:
this is not about sharing space, scenario is about listing permission of shared space.

Reply:
When step list the permission, I checked and this step for sharing space via link works